Physical_Engine/Query/Geometry.cs

```diff
@@ -59,6 +59,39 @@ public static oM.Geometry.ISurface Geometry(this oM.Physical.Elements.ISurface s
 
         /***************************************************/
 
+        [Description("Gets the geometry of a Pile as a single line. Method required for automatic display in UI packages.")]
+        [Input("pile", "Pile to get the line geometry from.")]
+        [Output("curve", "The curve defining the Pile.")]
+        public static IGeometry Geometry(this Pile pile)
+        {
+            return pile.IsNull() ? null : pile.Location;
+        }
```

Comment:
This method should not be needed, taken there is a method for `IFramingElement`

Reply:
I even used the `IFramingElement` method on the test script 🤦‍♂️